src/runtime/config/configuration-remote.toml.in

```diff
@@ -38,7 +38,7 @@ remote_hypervisor_timeout = 600
 # Each member of the list is a regular expression, which is the base name
 # of the annotation, e.g. "path" for io.katacontainers.config.hypervisor.path"
 # Note: Remote hypervisor is only handling the following annotations
-enable_annotations = ["machine_type", "default_memory", "default_vcpus", "image"]
+enable_annotations = ["machine_type", "default_memory", "default_vcpus", "image", "default_gpus", "gpu_model"]
```

Comment:
What does "default_gpus" / "gpu_model" mean ?

Reply:
@Apokleos the default_gpus is the number of gpus required in the Kata VM and likewise gpu_model is the model name. These will be used by the remote hypervisor implementation ie cloud-api-adaptor to select the right cloud instance for the POD. 
This document explains the resource management aspect of remote hyp. Although it's specific for CPU and Memory but the same concept applies for GPU - https://docs.google.com/presentation/d/1GWNgQdRC5WxrXz_0XCW3DGIfzQHkO4MaN-8BlRPuTDc/edit?usp=sharing

Reply:
Thx @bpradipt for your clarifications for this point. 
It almost looks good to me. waiting this PR to be opening.